csrc/mmdeploy/backend_ops/tensorrt/common_impl/nms/allClassNMS.cu

```diff
@@ -196,29 +200,18 @@ pluginStatus_t allClassNMS_gpu(cudaStream_t stream, const int num, const int num
       P(1), P(2), P(3), P(4), P(5), P(6), P(7), P(8), P(9), P(10),
   };
 
-  const int BS = 512;
   const int GS = num_classes;
   const int t_size = (top_k + BS - 1) / BS;
 
   ASSERT(t_size <= 10);
-  kernel[t_size - 1]<<<GS, BS, BS * t_size * sizeof(bool), stream>>>(
+  kernel[t_size - 1]<<<GS, BS, 0, stream>>>(
       num, num_classes, num_preds_per_class, top_k, nms_threshold, share_location, isNormalized,
       (T_BBOX *)bbox_data, (T_SCORE *)beforeNMS_scores, (int *)beforeNMS_index_array,
       (T_SCORE *)afterNMS_scores, (int *)afterNMS_index_array, flipXY);
 
   cudaError_t code = cudaGetLastError();
-  if (code != cudaSuccess) {
-    // Verify if cuda dev0 requires top_k to be reduced;
-    // sm_53 (Jetson Nano) and sm_62 (Jetson TX2) requires reduced top_k < 1000
-    auto __cuda_arch__ = get_cuda_arch(0);
```

Comment:
may also delete the definition of `get_cuda_arch`

Reply:
Not in this PR. RotatedNMS use get_cuda_arch too. And I am not sure if launch bounds works on rotated detection model.